python-checks-testkit/src/main/java/org/sonar/python/checks/utils/PythonCheckVerifier.java

```diff
@@ -56,18 +59,25 @@ public static void verify(String path, PythonCheck check) {
     createVerifier(file, check).assertOneOrMoreIssues();
   }
 
+  public static void verifyWithGlobals(String path, PythonCheck check, String packageName, Map<String, Set<Symbol>> globalSymbols) {
```

Comment:
This is a new method which would go in the public API.
An alternative to this method would be to allow testing multiple files together: I think that's what we do for COBOL. See MultiFileVerifier in https://github.com/SonarSource/sonar-analyzer-commons/tree/master/test-commons

Reply:
Thanks! I didn't know about MultiFileVerifier, I will use it!